<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>760</width>
     <height>760</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>760</width>
     <height>760</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>760</width>
     <height>760</height>
    </size>
   </property>
   <widget class="QLabel" name="cerceve">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 225, 151);
border-radius:140px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cerceve2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(131, 189, 117)
;
border-radius:140px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time_spent">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time spent:</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_username">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>76</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(53, 124, 60);
border-radius:40px;
</string>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(53, 124, 60);
border-radius:40px;</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
